exchange/exchange.go

```diff
@@ -197,6 +197,11 @@ type BidderRequest struct {
 }
 
 func (e *exchange) HoldAuction(ctx context.Context, r AuctionRequest, debugLog *DebugLog) (*openrtb2.BidResponse, error) {
+	reject := r.HookExecutor.ExecuteProcessedAuctionStage(r.BidRequestWrapper.BidRequest)
+	if reject != nil {
+		return nil, reject
```

Comment:
This line of code isn't covered by tests, can this be updated?

Reply:
It is covered on https://github.com/pi-prebid/prebid-server/pull/15/files#diff-03fbcce17cfcde19069516238b9b4bb8406dd50207915fc88bee6b1414cbfd73R4680 where we reject the request and verify the bid response (all further processing is canceled and the response should have an nbr code) if I understood you correctly

Reply:
I see, thanks for the response!